playbooks/deploy_harbor.yaml

```diff
@@ -0,0 +1,27 @@
+- hosts: localhost # Kubectl is configured from this machine
+  roles:
+    - deploy_DB_longhorn
+    - deploy_harbor
+  vars:
+
+    # Helm chart versions 
+    harbor_version: "1.10.2" 

```

Comment:
Could we put a URL for the release page of Harbor / longhorn in the comments to make it easy to lookup new versions